go.mod

```
@@ -1,14 +1,14 @@
 module github.com/adevinta/vulcan-local
 
-go 1.20
+go 1.21
```

```suggestion
go 1.21.0
```

> For example, 1.21 < 1.21rc1 < 1.21rc2 < 1.21.0 < 1.21.1 < 1.21.2.

Reference: https://go.dev/doc/toolchain#version